.vscode/launch.json

```diff
@@ -21,6 +21,24 @@
       ],
       "preLaunchTask": "npm: watch"
     },
+    {
+      "name": "Run Extension & vscode-microprofile",
```

Comment:
I'm also open to changing the name :

- "Run Extension with MicroProfile"
- "Run Quarkus & MicroProfile Extensions"
...

Reply:
What about `Run Extension with installed MicroProfile`?

Reply:
and the second like this (which is not a perfect english) `Run Extension with sources MicroProfile`

Reply:
I would think "installed" implies from the marketplace. I think "Run Extension with Development MicroProfile" or "Run Extension with MicroProfile Project" gets closer to what I'd like.

Reply:
I think "Run Extension with MicroProfile sources" could work.

Reply:
Installed means to install from vsix too, no?

Reply:
Yes. that as well. That's why I prefer the term "source" or "development". It makes it clear we're running the project folder and that no .vsix or marketplace content is consumed.